### DISCUSSION TITLE
Question: Is it possible modify the sensor to show the 10% of the total value showed?

### DISCUSSION THREAD

**Discussion by rafa16277** [tijsverkoyen/HomeAssistant-FusionSolar#214] [Q&A] [answered]
I have a shared installation, and I have the 10% of the total of the production. Is there any way to modify the value to show me that 10% of the total?

**Comment by tijsverkoyen** [comment]
You can create helpers in Home Assistant to achieve this.

**Reply by rafa16277** [reply]
Can you give me an example of how to do it?

**Reply by tijsverkoyen** [reply, marked as answer]
Settings → Devices & Services → Helpers → Create helper -> Template Helper → Template a sensor.

In the field state template enter something like below:

```
{{ states('sensor.xxx_current_year_consumption')|float / 100 * 10 }}
```

Offcourse change the sensor name to the one you want to calculate the value for.

**Reply by rafa16277** [reply]
Thank you so much. I will try this

**Reply by rafa16277** [reply]
Working as I wanted!!!!. Thank you so much
